ts/src/pro/bitpanda.ts

```diff
@@ -0,0 +1,1335 @@
+            this.tickers[symbol] = this.parseWSTicker (ticker);
+            this.tickers[symbol]['timestamp'] = this.parse8601 (datetime);
+            this.tickers[symbol]['datetime'] = datetime;
+            client.resolve (this.tickers[symbol], 'ticker.' + symbol);
+        }
+        client.resolve (this.tickers, 'tickers');
+    }
+
+    parseWSTicker (ticker, market = undefined) {
+        //
+        //     {
+        //         instrument: 'ETH_BTC',
+        //         last_price: '0.053752',
+        //         price_change: '-0.000623',
+        //         price_change_percentage: '-1.17',
+        //         high: '0.055',
+        //         low: '0.052662',
+        //         volume: '6.3821593247'
+        //     }
+        //
+        const marketId = this.safeString (ticker, 'instrument');
+        return this.safeTicker ({
+            'symbol': this.safeSymbol (marketId, market),
+            'timestamp': undefined,
+            'datetime': undefined,
+            'high': this.safeString (ticker, 'high'),
+            'low': this.safeString (ticker, 'low'),
+            'bid': undefined,
+            'bidVolume': undefined,
+            'ask': undefined,
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': undefined,
+            'close': this.safeString (ticker, 'last_price'),
+            'last': this.safeString (ticker, 'last_price'),
+            'previousClose': undefined,
+            'change': this.safeString (ticker, 'price_change'),
+            'percentage': this.safeString (ticker, 'price_change_percentage'),
+            'average': undefined,
+            'baseVolume': undefined,
+            'quoteVolume': this.safeNumber (ticker, 'volume'),
+            'info': ticker,
+        }, market);
+    }
+
+    async watchMyTrades (symbol: string = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name bitpanda#watchMyTrades
+         * @see https://developers.bitpanda.com/exchange/#account-history-channel
+         * @description get the list of trades associated with the user
+         * @param {string} symbol unified symbol of the market to fetch trades for. Use 'any' to watch all trades
+         * @param {int|undefined} since timestamp in ms of the earliest trade to fetch
+         * @param {int|undefined} limit the maximum amount of trades to fetch
+         * @param {object} params extra parameters specific to the bitpanda api endpoint
+         * @returns {[object]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
+         */
+        await this.loadMarkets ();
+        if (symbol !== undefined) {
+            const market = this.market (symbol);
+            symbol = market['symbol'];
+        }
+        await this.authenticate (params);
+        const url = this.urls['api']['ws'];
+        const messageHash = 'myTrades';
```

Comment:
@pcriadoperez why aren't we using the symbol to build the messageHash? 
how could we resolve two different watchMyTrades futures without it?
```
a = bitpanda.watchMyTrades('LTC/EUR')
b =  bitpanda.watchMyTrades('BTC/USDT')
```

Reply:
Great catch! I added a fix